cmd/fpga_tool/fpga_tool.go

```diff
@@ -34,9 +34,13 @@ const (
 
 func main() {
 	var err error
+
 	var bitstream, device string
+
 	var dryRun, force, quiet bool
+
 	var port uint
+
```

Comment:
I wonder why the check is so aggressive here. Probably the expected format is something like
```go
var (
        err error
        bitstream, device string
        dryRun, force, quiet bool
        port uint
)
```

Reply:
Thanks for the suggestion. Will try that.